src/cli.rs

```diff
@@ -198,12 +210,45 @@ pub fn run() -> Result<()> {
     )
     .wrap_err("Failed to parse config as TOML")?;
 
+    let current_path = env::current_dir().wrap_err("Failed to get current directory")?;
     let conf = Config {
         agenix: agenix_conf,
         root: conf_path,
     };
 
-    let current_path = env::current_dir().wrap_err("Failed to get current directory")?;
+    trace!("rekey_all? {}", opts.rekey_all);
+    if opts.rekey_all {
+        let mut paths = Vec::new();
+        for pathspec in &conf.agenix.paths {
+            for path in glob::glob(&pathspec.glob)
+                .wrap_err_with(|| format!("Failed to match glob pattern '{}'", &pathspec.glob))?
+            {
+                let path = path.wrap_err_with(|| {
+                    format!("Failed to iterate over glob pattern '{}'", &pathspec.glob)
+                })?;
+                paths.push(path);
+            }
+        }
+
+        for path in paths {
+            if let Err(e) = self::try_process_file(&conf, &path, &opts, &current_path) {
+                error!("Failed to rekey file '{}': {}", path.display(), e);
```

Comment:
I'm not totally satisfied with just printing the error, but at least we make it known to the user that something went wrong when rekeying that file. If anybody comes along later and wants a more robust solution, we can fix this then.

Reply:
Makes sense. Do you have a preferred solution? Rolling back the rekey processes if any file fails or rekeying in temporary a staging area (and maybe adding a `--keep-going` flag to get back something similar to the current behavior)? I am a bit conflicted about what feels like the "right thing" to do in that situation, but just logging the error does feel a bit insufficient.

Reply:
I actually think this is fine for the moment. It's at least more information than the shell script gives out.

If people run into issues, they can file a bug report and we can work on improving the situation at that point.